dizoo/atari/example/atari_dqn_dist_rdma.py

```diff
@@ -26,46 +28,53 @@ def main():
         set_pkg_seed(cfg.seed, use_cuda=cfg.policy.cuda)
 
         model = DQN(**cfg.policy.model)
-        policy = DQNPolicy(cfg.policy, model=model)
+
+        # Consider the case with multiple processes
+        if task.router.is_active:
+            # You can use labels to distinguish between workers with different roles,
+            # here we use node_id to distinguish.
+            if task.router.node_id == 0:
+                task.add_role(task.role.LEARNER)
+            else:
+                task.add_role(task.role.COLLECTOR)
+
+        logging.debug("label {}".format(task.router.labels))
+        logging.debug("task role {}".format(task._roles))
 
         if 'learner' in task.router.labels:
+            policy = DQNPolicy(cfg.policy, model=model)
             logging.info("Learner running on node {}".format(task.router.node_id))
             buffer_ = DequeBuffer(size=cfg.policy.other.replay_buffer.replay_buffer_size)
-            task.use(
-                context_exchanger(
-                    send_keys=["train_iter"],
-                    recv_keys=["trajectories", "episodes", "env_step", "env_episode"],
-                    skip_n_iter=0
-                )
-            )
-            task.use(model_exchanger(model, is_learner=True))
+            task.use(ContextExchanger(skip_n_iter=0))
+            task.use(ModelExchanger(model))
             task.use(nstep_reward_enhancer(cfg))
             task.use(data_pusher(cfg, buffer_))
             task.use(OffPolicyLearner(cfg, policy.learn_mode, buffer_))
             task.use(CkptSaver(cfg, policy, train_freq=1000))
 
         elif 'collector' in task.router.labels:
+            policy = DQNPolicy(cfg.policy, model=model)
             logging.info("Collector running on node {}".format(task.router.node_id))
             collector_cfg = deepcopy(cfg.env)
             collector_cfg.is_train = True
+            logging.info(cfg.env.manager)
+            logging.info(type(cfg.env.manager))
+            # task.router.judge_use_cuda_shm(cfg)
```

Comment:
Used  `task.router.judge_use_cuda_shm(cfg)` to detect whether the conditions for using CUDA-shared-tensor buffer are met.